addons/gamification/models/res_users.py

```diff
@@ -353,6 +362,20 @@ def get_gamification_redirection_data(self):
         self.ensure_one()
         return []
 
+    def action_log_interacted_throttled(self):
```

Comment:
does it need to be public ?

I would call it `_update_last_interacted_date` to match `_update_last_login`